src/Setup.elm

```diff
@@ -0,0 +1,166 @@
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view
+        , update = update
+        }
+
+
+init : Model
+init =
+    { startYear = 2017
+    , endYear = 2019
+    , einsteinFactor = 1
+    , districtFactor = 1
+    , playOffFactor = 1
+    , offSeasonFactor = 1
+    }
+
+
+stringToFloat : String -> Float
+stringToFloat number =
+    let
+        maybeFloat =
+            String.toFloat number
+    in
+    case maybeFloat of
+        Nothing ->
+            0
+
+        Just value ->
+            value
+
+
+yearsInput : Int -> Input.Label Msg -> (Float -> Msg) -> Element Msg
+yearsInput value label onChange =
+    el [ Element.centerX, Element.width <| px 345 ] <|
+        Input.slider
+            [ Element.height (Element.px 20)
+            , Element.behindContent
+                (Element.el
+                    [ Element.width Element.fill
+                    , Element.height (Element.px 6)
+                    , Element.centerY
+                    , Background.color <| rgb255 0 0 200
+                    , Border.rounded 4
+                    , Element.centerX
+                    ]
+                    Element.none
+                )
+            ]
+            { onChange = onChange
+            , label = label
+            , min = 2007
+            , max = 2019
+            , step = Just 1
+            , value = toFloat value
+            , thumb =
+                Input.defaultThumb
+            }
+
+
+factorInput : Float -> String -> (String -> Msg) -> Element Msg
+factorInput value labelText sendMsg =
+    el [ Element.alignRight, Element.width <| px 317 ] <|
+        Input.text [ Element.width <| px 60, Font.size 20 ]
+            { onChange = sendMsg
+            , placeholder = Nothing
+            , text = String.fromFloat value
+            , label = Input.labelLeft [ Element.centerY, Element.centerX, Font.size 30 ] <| Element.text labelText
+            }
+
+
+continueButton : Element Msg
+continueButton =
+    el [ Element.centerX, Element.centerY ] <|
+        Input.button
+            [ Background.color <| Element.rgb255 255 255 255
+            , Border.rounded 7
+            , Font.size 30
+            , Font.color <| Element.rgb255 0 0 200
+            , Border.width 4
+            ]
+            { onPress = Just Continue
+            , label = text "Continue"
+            }
+
+
+update : Msg -> Model -> Model
+update msg model =
+    case msg of
+        EinsteinFactor value ->
+            { model | einsteinFactor = value }
+
+        StartYear value ->
+            { model | startYear = value }
+
+        EndYear value ->
+            { model | endYear = value }
+
+        DistrictFactor value ->
+            { model | districtFactor = value }
+
+        PlayOffFactor value ->
+            { model | playOffFactor = value }
+
+        OffSeasonFactor value ->
+            { model | offSeasonFactor = value }
+
+        default ->
+            model
```

Comment:
Don't add default cases when you don't have to. You're just making it harder for the compiler to assist you.